react-app/src/app/screens/Game/components/Board/index.js

```diff
@@ -23,3 +24,12 @@ function Board({ squares, onClick }) {
 }
 
 export default Board;
+
+Board.propTypes = {
+  squares: PropTypes.arrayOf(PropTypes.string),
+  onClick: PropTypes.func.isRequired
+};
+
+Board.defaultProps = {
+  squares: Array(9).fill(null)
```

Comment:
`[]`? :thinking:

Reply:
Yes. I though the default prop needed to be an empty array (with length) in order to render properly. I just changed it to the way you suggest and it works perfectly.